### PR TITLE
Shell model node

### PR DESCRIPTION
**Pull request by TheDoctorRAB**
I changed the name of the file to index.md

### CODE REVIEW

File: shell-model/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+### Overview
+The most straightforward concept of the atom is the shell model, first proposed by Niels Bohr.
+
+A nucleus contains protons and neutrons. Protons carry a positive charge, and neutrons carry no charge. The nucleus is surrounded by shells of negatively-charged electrons.
+
+Each shell can only hold a fixed number of electrons, and each shell essentially represents a principal energy level. The electrons orbit around the nucleus. 
+
+(Quantum physics has shown this is more of an electron cloud, and there is a limit to how precise one can simultaneously know the position or momentum of a particle; aka the Heisenberg Uncertainty Principle. For now though, we are only concerned with the Bohr shell model.)
+
+Radioactive decay, therefore, is the spontaneous emission of different particles from an unstable (energy state) atom. This can be the emission of an electron, for example. The atom wants to achieve a stable state, but in the process may become charged due to the method of decay. Radioactivity then refers to the energy emitted as a result of decay.
```
Comment (gonuke) [inline shell-model/index.md]: Radioactive decay is largely a nuclear process.  Don't we need to introduce nuclear shell model in addition to electron shell model.

File: shell-model/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - none 
+learning_objectives: 
+  - reproduce a shell model of an atom 
+references:
+    - http://prezi.com/zmamtqeoiplg/?utm_campaign=share&utm_medium=copy 
+abet_outcomes: n/a
+assessments: 
+  - shell-model.yml 
+...
+# Radioactive Decay
```
Comment (gonuke) [inline shell-model/index.md]: Shouldn't the `title`, `uuid` and rendered title match better? These are three different topics

File: shell-model/index.md (hunk 1 above)
Comment (gonuke) [reply shell-model/index.md]: Not sure how the discussion of radioactive decay supports the learning objective.

File: shell-model/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: ATOMIC STRUCTURE 
+uuid: shell-model
+prerequisites:
+  - none
```
Comment (gonuke) [inline shell-model/index.md]: Maybe `periodic-table` is a good prerequisite?

**Review by gonuke — COMMENTED** [review]
I looked at this to consider the question of prerequisites but had a few other comments/questions

File: shell-model/index.md (hunk 2 above)
Comment (TheDoctorRAB) [reply shell-model/index.md]: I don't know what you mean. The title and uuid follow the guidelines.

Comment (gonuke) [reply shell-model/index.md]: The `title` is "Atomic Structure", the `uuid` is "shell-model", and the page title is "Radioactive Decay".  Those are at least 2 different topics.

File: shell-model/index.md (hunk 3 above)
Comment (TheDoctorRAB) [reply shell-model/index.md]: I'll add.

File: shell-model/index.md (hunk 2 above)
Comment (TheDoctorRAB) [reply shell-model/index.md]: This is one of the first nodes I did so I didn't know really what to do.

**Comment by gonuke** [comment]
Thanks @TheDoctorRAB - this looks good to me.... I'll ignore CI and merge...

**Pull request merged**